### DISCUSSION TITLE
How to export the docker-compose.yml if started with supabase cli ?

### DISCUSSION THREAD

**Discussion by dfang** [supabase/supabase#30441] [Questions]
How to export the docker-compose.yml if started with supabase cli ?

**Comment by silentworks** [comment]
There is no such option as exporting the docker-compose.yml. You would just dump your database data and follow the Supabase guide on Docker. https://supabase.com/docs/guides/self-hosting/docker

**Reply by dfang** [reply]
Yes, i'm self hosting supabase with this, but how to keep it the same as the supabase started by pnpm supabase start
